plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -313,6 +316,14 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 			return nil
 		}
 		return nil
+	case runTraceAgain:
+		// Get name of new traceflow
+		mostRecentTraceflow.Name = mostRecentTraceflow.Spec.Source.Pod + "-" + mostRecentTraceflow.Spec.Destination.Pod 
```

Comment:
Each case sets a different name, but they have the time in common, so wouldn't it make sense to only change the time?

Reply:
Yes, that works too. I suggested a common name generator function as this kind of code seems a bit redundant now, but of course not necessary to handle it in this PR. Up to you.

Reply:
Has the comment been addressed? My point was some traceflow requests' destination may be service or IP, so `mostRecentTraceflow.Spec.Destination.Pod` is empty. The new traceflow's name will be weird.

Reply:
The earlier way I had done it was to trim 15 characters off the end of the string, because those were guaranteed to have the date and time. I feel like that might be a better solution considering each traceflow name has a date and time.

Reply:
Yes, it works for me.